book.json

```diff
@@ -1,24 +1,18 @@
 {
-
-    "title": "Mac OS X Setup Guide ",
-
-    "links": {
```

Comment:
I think the link.home might be needed as we don't host the site at a root url

Reply:
Ok! Removing this commit then.